tyk-stack/values.yaml

```diff
@@ -213,7 +213,19 @@ tyk-gateway:
   gateway:
     # The hostname to bind the Gateway to.
     hostName: &gwHostName tyk-gw.local
-
+    # Enables validation of examples in the OAS spec. Defaults to false.
+    # It is used to set TYK_GW_OAS_VALIDATE_EXAMPLES
+    oasValidateExamples: false
```

Comment:
@andrei-tyk Can gateway and dashboard have different value for `oasValidateExamples` and `oasValidateSchemaDefaults`? Or they need to match?